### PR TITLE
WIFI-13126: feat: fixedconfig doc update

### PR DESCRIPTION
**Pull request by i-chvets**
# Description
Documentation update for new command.
Details are in:
https://telecominfraproject.atlassian.net/browse/WIFI-13126

# Summary of changes:
- Updated documentation.

### CODE REVIEW

File: PROTOCOL.md [hunk 1]
```diff
@@ -306,8 +306,40 @@ The device should answer:
          },
      "id" : <same number>
 }
+```
+
+#### Controller wants the device to apply a given fixed configuration
+
+Controller sends this command when it requires the device to apply fixed condfiguration, eg. country code. The device
```
Comment (carsten989) [inline PROTOCOL.md:313]: check spelling: condfiguration
also change to: ... applied and is running.

File: PROTOCOL.md [hunk 2]
```diff
@@ -306,8 +306,40 @@ The device should answer:
...
+should send message with `pending change` events until this version has been applied and running.
 
+```json
+{   "jsonrpc" : "2.0",
+    "method" : "fixedconfig",
+    "params" : {
+        "serial" : <serial number>,
+        "when" : Optional - <UTC time when to apply this config, 0 mean immediate, this is a suggestion>
```
Comment (carsten989) [inline PROTOCOL.md:321]: 0 means immediate

**Review by carsten989 — CHANGES_REQUESTED** [review]
a few spelling etc issues

File: PROTOCOL.md (hunk 2 above)
Comment (i-chvets) [reply PROTOCOL.md:321]: Done.

File: PROTOCOL.md (hunk 1 above)
Comment (i-chvets) [reply PROTOCOL.md:313]: Changed wording a bit to be in-sync with the code.

**Review by carsten989 — APPROVED** [review]

**Pull request merged**